learning_resources/site_scrapers/utils.py

```diff
@@ -5,6 +5,7 @@
 
 
 def scraper_for_site(url):
+    url = url.replace("http://", "https://")
```

Comment:
BTW `http://` is legal in query parameters, so theoretically this could change more than the protocol. (E.g., our login URL is `https://api.rc.learn.mit.edu/learn/login?next=https://rc.learn.mit.edu/`)

Very likely those sorts of URLs don't matter for this scraping (and maybe changing http to https in query params would be desirable, too).